### PR TITLE
move boot.img to output directory

### LINKED ISSUE #134: provide recovery images
There are some devices where one cannot use TWRP as recovery to flash lineageos4microg like sargo (pixel 3A). 
There is one solution described [here](https://github.com/lineageos4microg/docker-lineage-cicd/issues/105#issuecomment-700635488) how to use an official lineageos recovery and exchange keys to enable flashing a lineageos4micro. These manual steps would not be needed (also the many hours of fruitless online search to find this solution)

It would really help if the recovery images (boot.img) including the specific keys would already be provided per devices

### PR DESCRIPTION
**Pull request by thrillfall**
move (and rename) boot.img so it can be exposed on the downloads page
fixes #134

### CODE REVIEW

File: src/build.sh [hunk 1]
```diff
@@ -301,6 +301,8 @@ for branch in ${BRANCH_NAME//,/ }; do
             sha256sum "$build" > "$ZIP_DIR/$zipsubdir/$build.sha256sum"
           done
           find . -maxdepth 1 -name 'lineage-*.zip*' -type f -exec mv {} "$ZIP_DIR/$zipsubdir/" \; &>> "$DEBUG_LOG"
+	  recovery_name=$(echo "$build" | sed "s/.zip//")
```
Comment (spolack) [inline src/build.sh:304]: Please dont reuse the iteration variable from for loop, but rather build the name from scratch using build_date and codename.

File: src/build.sh [hunk 2]
```diff
@@ -301,6 +301,8 @@ for branch in ${BRANCH_NAME//,/ }; do
...
+find . -maxdepth 1 -name 'boot.img' -type f -exec mv {} "$ZIP_DIR/$zipsubdir/${recovery_name}-recovery.img" \; &>> "$DEBUG_LOG"
```
Comment (spolack) [inline src/build.sh:305]: Please fix indentation.

**Review by spolack — CHANGES_REQUESTED** [review]

File: src/build.sh (hunk 1 above)
Comment (thrillfall) [reply src/build.sh]: done. thanks for the hint

File: src/build.sh (hunk 2 above)
Comment (thrillfall) [reply src/build.sh]: done

**Comment by thrillfall** [comment]
@spolack thanks for your support on this change and providing lineageos4micro in general!
once there is a recovery image for every device i'll provide an MR for updating the install instructions (for non-twrp devices like sargo)

**Pull request merged**